lib/matplotlib/backend_bases.py

```diff
@@ -3195,3 +3205,474 @@ def zoom(self, *args):
     def set_history_buttons(self):
         """Enable or disable back/forward button"""
         pass
+
+
+class ToolEvent(object):
```

Comment:
Yes, go ahead.
I am trying to take the habit of a last coment `done` everywhere, so I know that even if the code is outdated, action was taken on the comment